plugins/modules/identity/ipa/ipa_dnszone.py

```diff
@@ -28,10 +28,13 @@
     type: str
   dynamicupdate:
     description: Apply dynamic update to zone
```

Comment:
While we're at it :)
```suggestion
    description: Apply dynamic update to zone.
```